gen/flights/flight.py

```diff
@@ -142,7 +142,7 @@ def __init__(
             x: X coordinate of the waypoint.
             y: Y coordinate of the waypoint.
             alt: Altitude of the waypoint. By default this is AGL, but it can be
-            changed to MSL by setting alt_type to "RADIO".
+            changed to MSL by setting alt_type to "RADIO". // Mags: Isn't this incorrect? Both the default and the explanation?
```

Comment:
The default looks to be RADIO (not AGL). And you don't get MSL if you change to RADIO.

Reply:
The default is set on L151 ("BARO", which means MSL). The comment here is indeed wrong. It should say `can be changed to AGL by setting alt_type to "RADIO"`.